pkg/install/v1alpha1/cstor_volume_0.7.0.go

```diff
@@ -224,6 +234,8 @@ spec:
       name: {{ .Volume.owner }}
       labels:
         openebs.io/persistent-volume: {{ .Volume.owner }}
+        openebs.io/fs-type: {{ .Config.FSType.value }}
+        openebs.io/lun: {{ .Config.Lun.value }}
```

Comment:
I think we should reconsider our choice of putting the values into labels. Annotations might be a better place for these.

Reply:
yes. this should be annotations!